frontends/p4/typeChecking/typeChecker.cpp

```diff
@@ -2137,8 +2142,10 @@ TypeInference::actionCall(bool inActionList,
         LOG2("Action parameter " << dbp(p));
         if (it == arguments->end()) {
             params->push_back(p);
-            if ((p->direction != IR::Direction::None) || !inActionList)
-                typeError("%1%: parameter %2% must be bound", actionCall, p);
+            if(findContext<IR::EntriesList>() == nullptr) {// no checking for table entries entries
```

Comment:
This looks wrong to me; you should be type-checking action calls here exactly as everywhere else.
The syntax and semantics for the actions part is exactly the same as for the default_action. So the code should be the same.